src/core/isSymbol.js

```diff
@@ -0,0 +1,9 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
```

Comment:
This should be 2018

Reply:
Hmmm I amended a commit that I don't think pushed. Will redo